suzieq/db/parquet/pq_coalesce.py

```diff
@@ -1,17 +1,18 @@
+import logging
 import os
+import re
+import tarfile
+from collections import defaultdict
 from datetime import datetime, timedelta, timezone
-import logging
-from typing import List
 from itertools import repeat
-import tarfile
+from typing import List
 
 import pandas as pd
+import pyarrow.dataset as ds  # put this later due to some numpy dependency
```

Comment:
If everything works as expected, this comment is outdated and would be better to delete it.